CollectionAsserts.cs

```diff
@@ -75,6 +136,61 @@ public static void All<T>(IEnumerable<T> collection, Action<T, int> action)
 				throw new AllException(idx, errors.ToArray());
 		}
 
+#if XUNIT_VALUETASK
+		/// <summary>
+		/// Verifies that a collection contains exactly a given number of elements, which meet
+		/// the criteria provided by the element inspectors.
+		/// </summary>
+		/// <typeparam name="T">The type of the object to be verified</typeparam>
+		/// <param name="collection">The collection to be inspected</param>
+		public static void Collection<T>(IEnumerable<T> collection)
+		{
+			// This overload is necessary when there are two overloads with a params if you want
```

Comment:
^ see comment here. Code for this method could be simplified, but I figured it would be best to copy the existing method and cut out references to `elementInspectors` instead for consistency.